ocs_ci/framework/conf/default_config.yaml

```diff
@@ -210,7 +210,7 @@ ENV_DATA:
   rhel8_template: 'rhel85_ocs4qe'
   # ACM HUB deployment
   deploy_acm_hub_cluster: false
-  default_acm_unreleased_image: "2.5.2-DOWNSTREAM-2022-07-29-15-56-49"
+  default_acm_unreleased_image: "2.7.0-DOWNSTREAM-2022-11-25-10-53-02"
```

Comment:
After conversation with Shylesh I got know that ODF 4.11 should be tested with ACM 2.6, ODF 4.12 with ACM 2.7. Should we move this value out of the default config to the specific ODF version config file? 

So can we move it to here: 
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/conf/ocs_version/ocs-4.12.yaml
?

@keesturam @prsurve WDYT?

Than we have pre defined matrix for ODF version and we will have this defined per version.

Reply:
Sure we can work on the same to tag specific versions of ACM and submariner with ocs release